build/media-suite_helper.sh

```diff
@@ -368,12 +368,16 @@ do_wget() {
     $notmodified && [[ -f $archive ]] && curlcmds+=(-z "$archive" -R)
     [[ $hash ]] && tries=3
 
+    if [[ -f $archive ]] && [[ $hash ]] && check_hash "$archive" "$hash"; then
```

Comment:
```suggestion
    if [[ -f $archive ]] && [[ -n $hash ]] && check_hash "$archive" "$hash"; then
```
right?

Reply:
Both, it seems that bash will treat brackets without a flag as -n

Better for readability with the n flag though

Reply:
Although it probably would help in cases where `$hash` contains a part valid for the brackets
```bash
$ [[ $asd ]]; echo $?
1
$ asd="-z 123"
$ [[ $asd ]]; echo $?
0
```